examples/k8s-bookinfo.yaml

```diff
@@ -56,8 +56,10 @@ spec:
           value: http://$(CONTROLLER_SERVICE_HOST):$(CONTROLLER_SERVICE_PORT)
         - name: A8_DISCOVERY_BACKEND
           value: kubernetes
-        - name: A8_SERVICE
-          value: gateway:none
+        - name: A8_KUBERNETES_POD_NAME
```

Comment:
How should the gremlin JSONs look like? I'm not clear as to how these tags are matched by Gremlin. Does it expect the service to contain exactly the tags specified in the recipe? Or them and optionally others?

Note that other services, except the gateway, have additional tags now (all Kubernetes labels are mapped to Amalgam8 tags). For example, productpage's nginx is launched with `A8_SERVICE=productpage:service=productpage,version=v1`.